examples/osgi-test-example-bnd/org.osgi.test.example.player.impl/.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```

Comment:
Remove this prefs file since this is not a maven example.

Reply:
removed